tests/data/api2Data.json

```diff
@@ -18,24 +18,33 @@
         }
       }
     ],
+    "HIJ.cc/-/Blind_Submission": [
```

Comment:
do you still need to keep these json files?

Reply:
Right now, the different files have data for different purposes but I could work on merging all of the files into the single `fakeData.json` file